test/LocatedTriplesTest.cpp

```diff
@@ -0,0 +1,173 @@
+//  Copyright 2023, University of Freiburg,
+//  Chair of Algorithms and Data Structures.
+//  Author: Hannah Bast <bast@cs.uni-freiburg.de>
+
+#include <gtest/gtest.h>
+
+#include "./util/IdTableHelpers.h"
+#include "./util/IdTestHelpers.h"
+#include "index/CompressedRelation.h"
+#include "index/IndexMetaData.h"
+#include "index/LocatedTriples.h"
+#include "index/Permutations.h"
+
+// TODO: Why the namespace here? (copied from `test/IndexMetaDataTest.cpp`)
```

Comment:
To avoid very strange ODR-violation bugs when another `.cpp` file also has a member `V` and the two are linked together. Linking the tests together is necessary for the code coverage to work.